core/Plugin/Controller.php

```diff
@@ -476,25 +476,14 @@ protected function getLastUnitGraphAcrossPlugins($currentModuleName, $currentCon
      */
     protected function getGraphParamsModified($paramsToSet = array())
     {
-        if (!isset($paramsToSet['period'])) {
-            $period = Common::getRequestVar('period');
-        } else {
-            $period = $paramsToSet['period'];
-        }
+        $period = isset($paramsToSet['period']) ? $paramsToSet['period'] : null;
```

Comment:
That one now misses the default:

```suggestion
        $period = $paramsToSet['period'] ?? Piwik::getPeriod();
```